examples/wip/viewport-animation-first-person-map/app.js

```diff
@@ -313,6 +313,7 @@ class Root extends Component {
           height={viewportProps.height}
           onViewportChange={this._onViewportChange}
           transitionDuration={transitionDuration}
+          transitionProps={['longitude', 'latitude', 'zoom', 'bearing', 'pitch', 'position']}
```

Comment:
Shouldn't this set of `transitionProps` be the default, i.e. no need to specify it here?

Reply:
When a prop is specified in 'transitionProps' array, it needs to be part of the viewport props. Given most of our examples and applications don't use 'position' yet I removed it. But eventually we can add it.